models/public/mozilla-deepspeech-0.8.2/mozilla-deepspeech-0.8.2.md

```diff
@@ -0,0 +1,138 @@
+   ```
+1. Run conversion from .pbmm ("memory-mapped" graph) to .pb (the common GraphDef):
+   ```
+   ./pbmm_to_pb.py <download_path>/deepspeech-0.8.2-models.pbmm <download_path>/deepspeech-0.8.2-models.pb
+   ```
+1. Run Model Converter to convert .pb to IR files:
+   ```
+   <openvino_path>/deployment_tools/open_model_zoo/tools/downloader/converter.py --name mozilla-deepspeech-0.8.2
+   ```
+1. Run conversion from .scorer (language model format in Mozilla DeepSpeech v0.7.x/0.8.x) to .kenlm (LM format used in v0.6.x and by us):
+   ```
+   ./scorer_to_kenlm.py <download_path>/deepspeech-0.8.2-models.scorer <download_path>/deepspeech-0.8.2-models.kenlm
+   ```
+
+## Specification
+
+| Metric                          | Value                                     |
+|---------------------------------|-------------------------------------------|
+| Type                            | Speech recognition                        |
+| GFlops per audio frame          | 0.0472                                    |
+| GFlops per second of audio      | 2.36                                      |
+| MParams                         | 47.2                                      |
+| Source framework                | TensorFlow\*                              |
+
+## Accuracy
+
+| Metric                       | Value      | Parameters                                                       |
+| ---------------------------- | ---------- | ---------------------------------------------------------------- |
+| WER @ LibriSpeech test-clean | 8.39%      | with LM, beam_width = 32, Python CTC decoder, accuracy checker   |
+| WER @ LibriSpeech test-clean | **6.13%**  | with LM, **beam_width = 500**, C++ CTC decoder, accuracy checker |
+| WER @ LibriSpeech test-clean | **6.15%**  | with LM, **beam_width = 500**, C++ CTC decoder, demo             |
+
+*NB*: beam_width=32 is a low value for a CTC decoder, and was used to achieve reasonable evaluation time with Python CTC decoder in Accuracy Checker.
+Increasing beam_width improves WER metric and slows down decoding.  Speech recognition demo has a faster C++ CTC decoder module.
+
+## Performance
+
+## Input
+
+### Original Model
+
+ * **Audio MFCC coefficients**, name: `input_node` , shape: [1x16x19x26], format: [BxNxTxC], where:
+
+    - B - batch size, fixed to 1
+    - N - `input_lengths`, number of audio frames in this section of audio
+    - T - context frames: along with the current frame, the network expects 9 preceding frames and 9 succeeding frames. The absent context frames are filled with zeros.
+    - C - 26 MFCC coefficients per each frame
+
+   See [`accuracy-check.yml`](accuracy-check.yml) for all audio preprocessing and feature extraction parameters.
+
+ * Number of audio frames, INT32 value, name: `input_lengths`, shape [1].
+
+ * LSTM in-state (*c*) and input (*h*, a.k.a hidden state) vectors. Names: `previous_state_c` and `previous_state_h`, shapes: [1x2048], format: [BxC].
```

Comment:
Split `previous_state_c` and `previous_state_h` layers in the separate list items.

Reply:
Done, everywhere in this file.